app/controllers/application_controller.rb

```diff
@@ -1,5 +1,9 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
```

Comment:
- [ ] The app suppose to direct you to the sign-in page when running the rails server because you suppose to authenticate the user before showing any data, Now the app is showing the users index page directly without signing in, So please fix that.
[Hint] You can authenticate the user by adding `before_action :authenticate_user!` here in Application Controller.

Reply:
I'm having a great day, I hope you too! .

I still defend that that's optional, those are just some guidelines to get started with devise. No necessarily you need to do everything step by step, especially when you intendly don't want that behaviour for your user.